lib/membrane/rtp/session_bin.ex

```diff
@@ -36,6 +36,25 @@ defmodule Membrane.RTP.SessionBin do
   In such a case, SessionBin will receive payloaded packets and work as a simple proxy just forwarding the packets (and decrypting them if necessary).
   Therefore it is possible to specify in newly added pads if payloaders/depayloaders should be used for the certain stream.
 
+  ## Padding
+  Addition and removal of a padding from RTP packets is handled by the RTP plugin.
+
+  To send a packet with a padding, one should include `padding_size` field in `Membrane.Buffer`'s metadata.
+
+  E.g.
+
+  ```elixir
+  metadata = %{rtp: %{padding_size: 20}}
+  ```
+
+  will result in adding 20 bytes of padding (in total i.e. including the last byte denoting padding size)
```

Comment:
Isn't it a little too specific? This last byte *is* part of the padding.

```suggestion
  will result in adding 20 bytes of padding
```